packages/expo-cli/src/commands/client/selectDistributionCert.js

```diff
@@ -39,8 +39,9 @@ async function selectDistributionCert(context, options = {}) {
   if (distributionCert === 'GENERATE') {
     distributionCert = await generateDistributionCert(context);
   } else if (distributionCert === 'UPLOAD') {
-    distributionCert = (await promptForCredentials(context, ['distributionCert']))[0]
-      .distributionCert;
+    const userProvidedCredentials = await promptForCredentials(context, ['distributionCert']);
+    distributionCert = userProvidedCredentials[0].distributionCert;
```

Comment:
Please do not override existing variables, especially when the values have totally different types. It may cause a lot of headaches.

Reply:
makes sense. theres an analogous file for push keys that does smth similar so i've gone and propagated the changes there as well.